### PR TITLE
DOC-3976-CD-Release-Notes-1.25.x

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
CD Release Notes 1.25.x

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Review by krishi0408 — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -49,6 +49,32 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+#### Fixed issues
+- ECS service used to get deleted after first/second deployment in ECS Blue Green Deployment(CDS-91499, ZD-57892)
+   The issue is fixed by adding a condition for active service status.
+- The chained pipeline's input section in the Harness UI did not have the option to fetch manifests from the Git store for runtime config fields. (CDS-91387, ZD-57687)
+   This issue is fixed now and you should be able to set the defaults correctly.
+- Codeblocks embedded in Support AI replies were not rendering correctly. (CDS-91364)
+   This issue is fixed now.
+- In step details panel one stage moved the step details panel for other stages as well.(CDS-91351)
+   The steps graph in the execution view has been updated to auto-reset its position when they are dragged out of the visible area of the UI.
```
Comment (SudarshanPurohit) [inline release-notes/continuous-delivery.md:62]: The steps graph in the Execution View is now updated to auto-reset its position when a different stage is selected.

Comment (roshnisarangadharan) [inline release-notes/continuous-delivery.md:62]:
```suggestion
   The steps graph in the Execution View is now updated to auto-reset its position when a different stage is selected.
```

**Review by SudarshanPurohit — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -49,6 +49,44 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 ## February 2024
 
+### Version 1.25.5
+
+#### Fixed issues
+- ECS services got deleted after the first or second deployment in th ECS Blue Green deployment. (CDS-91499, ZD-57892)
```
Comment (roshnisarangadharan) [inline release-notes/continuous-delivery.md:55]:
```suggestion
- ECS services got deleted after the first or second deployment in the ECS Blue Green deployment. (CDS-91499, ZD-57892)
```

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Pull request merged**